tox.ini

```diff
@@ -1,5 +1,6 @@
 [tox]
-envlist = py27,pypy,py34,py35
+#envlist = py27,pypy,py34,py35,py36,py37
+envlist = py37
```

Comment:
This has a bad smell to it. It looks by default, running "tox" on the command line will only test py37. In CI, we'll test multiple python versions because we're explicitly passing them though?

At the very least, we should be testing py27 and py37 by default in tox.ini. Please also get rid of the comment, or change it something more human readable. In its current form, it's not clear whether or not we support those other pythons, or if we just forgot to remove a commented out line.

Reply:
Whops this was debugging cruft from an earlier version, the commented out line is what I want here... (I'll fix)